lib/Ohmbrewer_Screen.cpp

```diff
@@ -456,9 +456,9 @@ unsigned long Ohmbrewer::Screen::captureButtonPress() {
         displayStatusUpdate(status);
     }
     
-//    Serial.print("X = "); Serial.print(p.x);
-//    Serial.print("\tY = "); Serial.print(p.y);
-//    Serial.print("\tPressure = "); Serial.println(p.z);
+    Serial.print("X = "); Serial.print(p.x);
```

Comment:
I can comment these out again, but I was thinking that since we're doing a lot of touchscreen work we might just leave them as-is for now.